resourcematch-api/main.py

```diff
@@ -148,11 +161,21 @@ def search_responses_to_urls(search_responses):
             page_uuid_partition = page_uuid.partition('@')[0]
             page_data = get_book_page_data(book_uuid, page_uuid_partition)
             page_url_main = page_data['urls']['main']
+            page_book_title = decode_html_entities(page_data['book']['title'])
+            page_chapter_title = decode_html_entities(
```

Comment:
Just so we get the naming right, this is not actually always the chapter title. As an example, see the World History 2 [collection.xml](https://github.com/openstax/osbooks-world-history/blob/main/collections/world-history-volume-2.collection.xml). There are two subcollections before you get to actual content (e.g. `Connections Across Continents, 1500–1800` -> `Understanding the Past` -> then a leaf title `Developing a Global Perspective` such as [here](https://github.com/openstax/osbooks-world-history/blob/6d1ce8c0e206d02058eee24d840569017d57cbeb/modules/m00050/index.cnxml#L4). 

If you look at the page metadata for that [here](https://orn.openstax.org/orn/book:page/685e3163-1032-4529-bb3a-f97a54412704:ce80e622-4227-4d63-b296-dded349aabc8.json),  you'll see the various levels reflected. Maybe `page_data['contextTitles'][-2]` would work more often if we really want to get the parent of the leaf since the number of layers probably varies by book, or maybe we name this something more generic if we want to grab this value to reflect it may not always be chapter :man_shrugging: .